parm/parm_fv3diag/diag_table

```diff
@@ -22,6 +22,7 @@
 "gfs_dyn",     "ps",          "pressfc",      "fv3_history",    "all",  .false.,  "none",  2
 "gfs_dyn",     "hs",          "hgtsfc",       "fv3_history",    "all",  .false.,  "none",  2
 
+"gfs_phys",     "cldfra",       "cldfra",        "fv3_history",    "all",  .false.,  "none",  2
```

Comment:
What will happen if we try to run GFDL MP with this new variable? Right now cycling still only works with GFDL.

Reply:
> What will happen if we try to run GFDL MP with this new variable? Right now cycling still only works with GFDL.

@WalterKolczynski-NOAA This should have no impact on cycling with GFDL MP, but I have not tested.

Reply:
This particular change breaks cycling (hybrid EnVar) when trying to compute ensemble mean from the surface files.
The `sfcfHHH.nc` file now contains a 4D variable `cldfra`.  I am not sure if it is a consequence of this line or elsewhere.  I apologize if it was introduced elsewhere.  

![Screen Shot 2022-07-16 at 10 53 01 AM](https://user-images.githubusercontent.com/11394126/179360015-40eb5f9c-8905-4eb2-bc90-3b4908ea0731.png)

As a result, a change in needed in the GSI utility computing surface ensemble mean.  Some basic debugging revealed [this](https://github.com/NOAA-EMC/GSI-utils/blob/develop/src/EnKF/gfs/src/getsfcensmeanp.fd/getsfcensmeanp.f90) to skip `cldfra`.
If it is not skipped, the utility fails during the `epos` step in the workflow.
![Screen Shot 2022-07-16 at 10 52 32 AM](https://user-images.githubusercontent.com/11394126/179359998-8d5e785f-408e-4c91-860a-99547efcb5ac.png)

A test is running with the change in the GSI utility to skip this variable.
The test branch of GSI-utils is [here](https://github.com/NOAA-EMC/GSI-utils/compare/bugfix/getsfcensmeanp_cldfra?expand=1)

The test is being performed as part of PR #859.

Q. for possibly @yangfanglin and @RussTreadon-NOAA.
Should the variable be skipped or should the variable be averaged? 
The test above is skipping it (easy). 
To compute the average with this code will need work.

FYI. @WalterKolczynski-NOAA

Reply:
Add @emilyhcliu since the question at hand pertains to cloud fraction, `cldfra`.

Reply:
> Q. for possibly @yangfanglin and @RussTreadon-NOAA. Should the variable be skipped or should the variable be averaged? The test above is skipping it (easy). To compute the average with this code will need work.
> 

@aerorahul   I think cldfra can be skipped for computing ensemble mean surface file.  @emilyhcliu if a cycled experiment is running with GFDL MP, cld_amt is the actual cloud fraction and is saved in atmf .  If running with Thompson MP, cldfra is the cloud fraction and saved in sfcf.   However, my impression is that even cld_amt was not used in any part of the DA in GFS.v16.

Reply:
Question for @yangfanglin .  Why does Thompson MP write cldfra to sfcfXXX?   It seems the 3d cloud fraction field should go in atmfXXX.  A 3d field is not a surface field.

Reply:
@RussTreadon-NOAA 
I was about to ask the same question.
`cldfra` is neither a "surface" field, nor a `2D` field.

Reply:
> @RussTreadon-NOAA I was about to ask the same question. `cldfra` is neither a "surface" field, nor a `2D` field.

@aerorahul @RussTreadon-NOAA The model has two groups of outputs. dyn/atmf and phy/sfc.  For GFDL MP, cld_amt is a "prognostic" variable in the dycore group;  For Thompson MP, cldfra is a diagnostic variable from CCPP physics in the phy/sfc output group.  Placing variables across group is technical difficult, if no impossible.  This is the major reason the 3d cldfra is written in sfc group.  I had long discussion with Jun (@junwang-noaa ) about this, but this is the best we can do given the current model infrastructure.    Please note @ChunxiZhang-NOAA is going to make a minor change to the diag_table to have cldfra labeled as fv3_history2d